src/app/organizations/sponsorships/families-for-enterprise-setup.component.ts

```diff
@@ -40,7 +40,7 @@ import { OrganizationPlansComponent } from 'src/app/settings/organization-plans.
     templateUrl: 'families-for-enterprise-setup.component.html',
 })
 export class FamiliesForEnterpriseSetupComponent implements OnInit {
-    @ViewChild(OrganizationPlansComponent, { static: false }) 
+    @ViewChild(OrganizationPlansComponent, { static: false })
```

Comment:
Very minor, but missing semicolon on this line.

Reply:
damn running linter fixes! ;)

Thanks for the manual double-check

Reply:
oops, this is actually at the top-level in the class -- no semicolon